routes/index.js

```diff
@@ -6,4 +6,8 @@ router.get("/", (req, res) => {
     res.render("index");
 });
 
+// router.get("/user-data", (req, res) => {
```

Comment:
Please remove commented code or provide a reason as to why this is still in the repo (e.g. by adding a TODO)